front/app/containers/Admin/settings/customize/Header/HeaderImageDropzone.tsx

```diff
@@ -72,10 +72,13 @@ const HeaderImageDropzone = ({
     return ratio;
   };
 
+  const theme: any = useTheme();
   const overlayColor =
-    latestAppConfigStyleSettings?.signedOutHeaderOverlayColor;
+    latestAppConfigStyleSettings?.signedOutHeaderOverlayColor ||
+    theme.colorMain;
   const overlayOpacity =
-    latestAppConfigStyleSettings?.signedOutHeaderOverlayOpacity;
+    latestAppConfigStyleSettings?.signedOutHeaderOverlayOpacity ||
+    theme.signedOutHeaderOverlayOpacity * 100;
```

Comment:
This will show the default color + overlay in the admin settings, but not on the landing page when the value hasn't been saved.

Also, in the back-office, we'd have to change a value before we could save this.

The obvious downside to this solution is that it could be confusing to developers why they don't see the overlay on the landing page while it seems saved here.

I'm thinking of other solution... What if we we deleted the default header image in dev mode?

Reply:
> not on the landing page when the value hasn't been saved.

Actually, if the value hasn't been saved, the same default value will be used https://github.com/CitizenLabDotCo/cl2-component-library/blob/master/src/utils/styleUtils.ts#L528
So, the back office and the landing page look the same if the value is blank.

Also, the issue affects not only development but production too.

Reply:
Sorry, I'm not deep enough into this. What specifically are you asking for? Could you summarize the solution?

And could you check that theme.signedOutHeaderOverlayOpacity * 100 converts to a correct value in any case? It needs to be between 0 and 1. So not sure why we do * 100.

Reply:
CSS uses the value from 0 to 1. And we convert it to 0-100 for inputs.